### DISCUSSION TITLE
Seq Log Forwarder

### DISCUSSION THREAD

**Discussion by R4cOOn** [datalust/seq-tickets#1526] [Help] [answered]
Hi,

We're using a licensed Seq hosted in an Azure Kubernetes cluster.
The running costs are getting quite high due to the demands in RAM and drive.
We're a small startup and the Seq instance requires more resources than our entire applications put together.

We'd like to host the Seq instance locally.
We would need something like a log forwarder that lives in the Kubernetes cluster, takes in all the logs and sends them to our local machine. It would need to keep a local copy in case of connectivity drop etc.

There's nothing like that in the current Seq documentation.
What looks like a potential solution is https://github.com/datalust/seq-app-jsonarchive 
It doesn't look like it can be fully integrated to do what I want without some effort.

In the previous Seq version documentation there's a mention of https://github.com/datalust/seq-forwarder
It does seem to fit my use case but I'm not sure it's still working with the current Seq.
Is it still supported? If it is, why isn't it mentioned in the documentation?

Any suggestion welcome.

**Comment by nblumhardt** [comment, marked as answer]
Hi! Thanks for your message.

Seq Forwarder is alive, and is worth trying out. It's not very widely-used, but even so, there don't seem to have been many problems raised with it. Your data volume is probably the biggest unknown factor that will determine how successful it is.

Regarding your current Seq instance in AKS, though - would you be able to share how it is sized, and what you're currently doing (with retention policies and input leveling/filtering) to control log volumes? Setting up signals for querying can also make a big difference, if you're not already doing that. It's often possible to tune things and reduce hardware requirements - if you'd like us to take a closer look, sending some more info to `support@datalust.co`, and possibly a diagnostic report from Settings > Diagnostics, will get the ball rolling.

HTH!
Nick

**Reply by R4cOOn** [reply]
Thanks for the answer.

I'll go and send you the details you ask.

Cheers.
